experimental/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ All notable changes to experimental packages in this project will be documented
 ### :boom: Breaking Change
 
 * feature(views): move views registration to MeterProvider constructor [#3066](https://github.com/open-telemetry/opentelemetry-js/pull/3066) @pichlermarc
+* feat(sdk-metrics-base): distinguish between Sum and Gauge in MetricData [#3079](https://github.com/open-telemetry/opentelemetry-js/pull/3079) @pichlermarc
```

Comment:
This doesn't really describe what the break was. Adding 2 values to the enum isn't really breaking and it doesn't look like you removed the old SINGULAR

Reply:
It also looks like you removed the jsdocs for SINGULAR. Did you mean to remove it and just forgot?

Reply:
Actually meant to remove it. Removed that in [d9de8d3](https://github.com/open-telemetry/opentelemetry-js/pull/3079/commits/d9de8d3a19581ba782f468ff9290a4c662d00ab1) and added some more details in [e920c78](https://github.com/open-telemetry/opentelemetry-js/pull/3079/commits/e920c7856d6581179610647903ecc048dbb11646). :slightly_smiling_face: